bowtie/report/src/components/AccordionSvg.js

```diff
@@ -0,0 +1,36 @@
+const AccordionSvg = ({ lines, implementationImage }) => {
+  var testCase;
+  lines.map((element) => {
+    const propertyObjectArray = Object.keys(element);
+    if (propertyObjectArray[0] == "implementation") {
+      if (element.implementation == implementationImage) {
+        if (element.results && element.expected) {
+          for (let i = 0; i < element.results.length; i++) {
+            if (element.results[i].valid !== element.expected[i]) {
+              testCase = "failed";
+            }
+          }
+        }
+      }
+    }
+  });
+
+  return (
+    testCase == "failed" && (
+      <div>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="16"
+          height="16"
+          fill="currentColor"
+          className="bi bi-check-circle-fill"
+          viewBox="0 0 16 16"
+        >
+          <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z" />
```

Comment:
Maybe all SVG icon definition like this could be put into separate files and then imported in JS code? I remember that there was some common solution for that

Reply:
Ya I have kept them in a separate svg folder initially, but there was an issue in mode toggling. --- I stored svgs in .svg file and but in that case, I had to use the path as **value** to **src** property. And so, the SVG is now an image and we can't add value of `fill` attribute dynamically  based on the current theme. (means we have to use two different svgs --- I was trying to use only one). 

Btw, there are other ways too. I'll do that!

Reply:
I remember there were also whole icon packs (used as any other dependencies) which could be imported in JS. I'm guessing we don't use any custom icons?